README.md

```diff
@@ -126,12 +162,12 @@ and keys in shared memory, then set variable with the zone name.
   }
   ```
 
-* Location, that when requested, inspects the stored certificate (if present) and will request a new certificate if necessary. The included `docker-compose.yml` shows how to use a `healthcheck:` configuration for the NGINX service to periodically request this endpoint.
-    ```nginx
-    location = /acme/auto {
-      js_content acme.clientAutoMode;
-    }
-    ```
+* Location, that when requested, inspects the stored certificate (if present) and will request a new certificate if necessary. The included `docker-compose.yml` shows how to use a `healthcheck:` configuration for the NGINX service to periodically request this endpoint.  This should usually be restricted to local or Docker network requests to avoid it being abused.
```

Comment:
let's add a link tp [ngx_http_access_module](http://nginx.org/en/docs/http/ngx_http_access_module.html) module so people can use it to protect the location.

Reply:
Done. Good suggestion 👍